testify/test_case.py

```diff
@@ -66,12 +68,8 @@ def _cmp_str(instance):
         """Return a canonical representation of a TestCase for sorting and hashing."""
         return "%s.%s" % (instance.__module__, instance.__name__)
 
-    def __cmp__(cls, other):
```

Comment:
i ask because the docstring mentions _sorting_, and preserving sort order seems like a useful thing to be doing  :)